content/purgatorio/now-go-build/a-messy-real-world.md

```diff
@@ -1,5 +1,185 @@
 ---
 title: A Messy Real World
 author: clone95
-description: ---
----
\ No newline at end of file
+description: This guide aims to show you the challenges and the messiness of real-world problems, and how to tackle them pragmatically. 
```

Comment:
> messiness

Maybe the "reality" and reality is a better or broader term (more generic) to use as opposed to calling the real world messy (more specific) - there are pros and cons of the way the real world is.

Reply:
ok will think about this